docs/source/cpp/compute.rst

```diff
@@ -198,6 +198,10 @@ the input to a single output value.
 +---------------+-------+-------------+------------------------+----------------------------------+-------+
 | mean          | Unary | Numeric     | Scalar Decimal/Float64 | :struct:`ScalarAggregateOptions` |       |
 +---------------+-------+-------------+------------------------+----------------------------------+-------+
+| max           | Unary | Numeric     | Scalar Struct          | :struct:`ScalarAggregateOptions` |       |
```

Comment:
"max" before "mean"

Reply:
Thanks. I also fixed the output type column.